samples/host_exerciser/host_exerciser.h

```diff
@@ -322,9 +309,8 @@ class host_exerciser : public test_afu {
     app_.add_option("-d,--delay", he_delay_, "Enables random delay insertion between requests")->default_val(false);
 
     // how many requests to interleave when TestMode is Throughput
-    app_.add_option("--interleave", he_interleave_, "number of interleave in Throughput mode \
-           {rdwrrdwr, rdrdwrwr, rdrdrdrdwrwrwrwr}")
-          ->transform(CLI::CheckedTransformer(he_interleave))->default_val("rdwrrdwr");
+    app_.add_option("--interleave", he_interleave_, "number of interleave in Throughput mode {0, 1, 2}")
+       ->default_val(0);
```

Comment:
There's no decoding of the numeric values in the help and/or documentation.

Reply:
decoding of the numeric values in HE-LPBK documentation  ( Markdown file )

Reply:
Still isn't clear. Please change the help message to provide more information (including the three patterns). The help message doesn't even start with a capital letter and the one before it does. Please make it clear (by providing  and consistent with other help messages.

Reply:
app_.add_option("--interleave", he_interleave_, "Interleave requests pattern in Throughput mode {0, 1, 2} \
        Interleave pattern:0 rd-wr-rd-wr  Interleave pattern:1 rd-rd-wr-wr \
        Interleave pattern:2 rd-rd-rd-rd-wr-wr-wr-Wr ")
       ->default_val(0);

Updated interleave help

Reply:
Line is too long. How about using a multiline raw string literal?
It could look something like this:

```c++
const char *interleave_help = R"desc(Interleave requests pattern to use in throughput mode {0, 1, 2}
indicating one of the following series of read/write requests:
0: rd-wr-rd-wr
1: rd-rd-wr-wr
2: rd-rd-rd-rd-wr-wr-wr-wr)desc";

class host_exerciser : public test_afu {
public:
    host_exerciser()
  : test_afu("host_exerciser")
  , count_(1)
  {
    // Mode
    app_.add_option("-m,--mode", he_modes_, "host exerciser mode {lpbk,read, write, trput}")
      ->transform(CLI::CheckedTransformer(he_modes))->default_val("lpbk");

    // Cache line
    app_.add_option("--cls", he_req_cls_len_, "number of CLs per request{cl_1, cl_2, cl_3, cl_4}")
       ->transform(CLI::CheckedTransformer(he_req_cls_len))->default_val("cl_1");

    // Configures test rollover or test termination
    app_.add_option("--continuousmode", he_continuousmode_, "test rollover or test termination")->default_val(false);

    // Delay
    app_.add_option("-d,--delay", he_delay_, "Enables random delay insertion between requests")->default_val(false);

    // Configure interleave requests in Throughput mode
    app_.add_option("--interleave", he_interleave_, interleave_help)->default_val(0);
  }

```

Reply:
Thank you Rojo  for review suggestions .Updated interleave help